apis/r/R/SOMADenseNdArray.R

```diff
@@ -0,0 +1,144 @@
+#' SOMADenseNdArray
+#'
+#' @description
+#' `SOMADenseNdArray` is a dense, N-dimensional array of `primitive` type, with
+#' offset (zero-based) `int64` integer indexing on each dimension with domain
+#' `[0, maxUint64)`. The `SOMADenseNdArray` has a user-defined schema, which
```

Comment:
You cannot have [0, maxUint64]  with an `integer64` as it is signed.

Reply:
Thanks! A leftover from before we updated the spec to replace uint64 w/ int64.